### ISSUE TITLE
Fontawesome is using math.random, which is reported by Fortify Code Scanner as a High Vulnerability issue

### ISSUE THREAD

**Issue by xuzhiqingnova** [FortAwesome/Font-Awesome#17051]
Fortify Code Analyzer the Math.random() function used in fontawesome.js as a high vulnerability issue (Abstract: Standard pseudorandom number generators cannot withstand cryptographic attacks.) and fortify recommend to use cryptographic PRNG such as window.crypto.random() function.

**Comment by robmadole** [comment]
Yes, we aren't using this for anything other than generating a unique ID. Using window.crypto seems overkill.
